tf/env/local/secrets-api.tf

```diff
@@ -29,13 +29,14 @@ resource "random_password" "api-app-jwt-secret" {
 }
 
 resource "kubernetes_secret" "api-app-secrets" {
+  for_each = toset(["default", "api-jobs"])
   metadata {
-    name = "api-app-secrets"
-    namespace = "default"
+    name      = "api-app-secrets"
+    namespace = each.value
```

Comment:
While `terraform` can `plan` this change properly, it seems dropping and recreating a secret like this _sometimes_ creates race conditions in Kubernetes, making tf fail at apply-time. Not sure what to do about it right now to be honest.

Reply:
This can be fixed using `moved` blocks.